brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPTableViewController.swift

```diff
@@ -63,42 +63,57 @@ class VIPTableViewController<C: CoordinatableRoutes,
     override func setupSubviews() {
         super.setupSubviews()
         
-        topInsetValue = sceneLeftAlignedTitle == nil ? 0 : (Margins.large.rawValue * 2) + 28
-        
-        view.addSubview(leftAlignedTitleLabel)
-        leftAlignedTitleLabel.snp.makeConstraints { make in
-            make.top.equalTo(view.safeAreaLayoutGuide.snp.top)
-            make.height.equalTo(topInsetValue)
-            make.leading.trailing.equalToSuperview().inset(Margins.large.rawValue)
-        }
+        topInsetValue = sceneLeftAlignedTitle == nil ? 0 : Margins.extraHuge.rawValue + 28
         
         view.addSubview(tableView)
         tableView.snp.makeConstraints { make in
             make.edges.equalToSuperview()
         }
         
+        tableView.addSubview(leftAlignedTitleLabel)
```

Comment:
amm.. whats this?

Reply:
<img width="689" alt="image" src="https://user-images.githubusercontent.com/99484553/174546983-0315f972-b05a-49d8-aa28-c62b6aec239f.png">

Reply:
id use large titles for this crap.. imo thats what the designers tried to replicate

Reply:
https://images.app.goo.gl/5bwRFvVoLuvGvjCn9

Reply:
Talked to Tina. Apparently they want it as it is in the designs. Also, sometimes, there is a button in front of that title. Seemed easier this way but can be changed.